p2p/libp2p/netMessenger_test.go

```diff
@@ -132,7 +132,10 @@ func containsPeerID(list []p2p.PeerID, searchFor p2p.PeerID) bool {
 func TestNewMemoryLibp2pMessenger_NilContextShouldErr(t *testing.T) {
 	netw := mocknet.New(context.Background())
 
-	mes, err := libp2p.NewMemoryMessenger(nil, netw, discovery.NewNullDiscoverer())
+	ctx := context.Background()
```

Comment:
please replace the three lines with variable declaration

Reply:
done.